timescaledb/how-to-guides/continuous-aggregates/create-a-continuous-aggregate.md

```diff
@@ -142,14 +142,31 @@ queries to run efficiently.
 
 </procedure>
 
+## Create a continuous aggregate on JOINed tables
+
```

Comment:
I am finding this term JOINed tables a bit odd/unclear, excuse me if it is a general term used by customers or user community. I would have preferred following wording for this, perform joins in the definition of continuous aggregates, or perform joins while creating/defining continuous aggregates, something like that.

Reply:
Agreed, I think I'd prefer `Create a continuous aggregate with a JOIN in the definition`

Reply:
I fiddled with the wording a bit. Is it clearer now?